kuma/static/styles/minimalist/components/_search-widget.scss

```diff
@@ -30,4 +41,92 @@
             box-shadow: none;
         }
     }
+
+    .toggle-form {
+        border: inherit;
+        height: inherit;
+        position: inherit;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        overflow: hidden;
+        width: $mobile-search-height;
+        top: $mobile-search-height;
+        right: 0;
+
+        span {
+            @include visually-hidden;
+        }
+
+        .search-icon,
+        .close-icon {
+            transition: transform 0.2s ease-in;
+            width: $icon-size;
+            height: $icon-size;
+            position: absolute;
+        }
+
+        .close-icon {
+            transform: translateY(-$mobile-search-height);
+        }
+    }
+
+    &.show-form {
+        position: fixed;
+        z-index: 1;
+
+        form {
+            background: $neutral-100;
+            transform: translateY($mobile-search-height);
+        }
+
+        .toggle-form {
+            top: 0;
+
+            .close-icon {
+                transform: translateY(0);
+            }
+
+            .search-icon {
+                transform: translateY($mobile-search-height);
+            }
+        }
+    }
+}
+
+@media #{$mq-tablet-and-up} {
+    .header-search {
+        position: inherit;
+        width: inherit;
+        height: 40px;
+        min-width: 80px;
+
+        &.show-form {
+            width: auto;
+            position: inherit;
+        }
+
+        // needs [role='search'] to make it more specific in order to
+        // override form.show-form styles
+        form[role='search'] {
+            @include reset-visually-hidden();
+            transition: none;
+            transform: translateY(0);
+            padding: 2px 8px;
+            border: 2px solid $text-color;
+            border-radius: 20px;
+            height: 40px;
+            min-width: 80px;
+
+            .search-icon {
+                display: inline-block;
+                flex: 0 0 25px;
+                fill: $text-color;
+            }
+        }
+
+        .toggle-form {
+            display: none;
+        }
```

Comment:
I prefer that we keep these in context so, for example, I would move this to the block at line 45 like so:

```
.toggle-form {
    border: inherit;
    height: inherit;
    position: inherit;
    display: flex;
    align-items: center;
    justify-content: center;
    overflow: hidden;
    width: $mobile-search-height;
    top: $mobile-search-height;
    right: 0;

    @media #{$mq-tablet-and-up} {
        display: none;    
    }
}
```

And then the same with the other instances such as `header-search` etc.

Reply:
Thanks Schalk! Updated and ready for another look 👀